src/orion/core/evc/experiment.py

```diff
@@ -106,10 +108,10 @@ def children(self):
         if not self._children and self._no_children_lookup:
             self._no_children_lookup = False
             query = {'refers.parent_id': self.item.id}
-            selection = {'name': 1}
+            selection = {'name': 1, 'version': 1}
             experiments = Database().read('experiments', query, selection=selection)
             for child in experiments:
-                self.add_children(ExperimentNode(name=child['name']))
+                self.add_children(ExperimentNode(name=child['name'], version=child['version']))
```

Comment:
Same for children

Reply:
Maybe because we never tried with an experiment tree with multiple versions of the same experiment? I don't know.